theme/static/js/hs_file_browser.js

```diff
@@ -646,9 +647,8 @@ function bindFileBrowserItemEvents() {
                 }
             }
         }
-
         updateSelectionMenuContext();
-    });
```

Comment:
Is removing the semicolon intentional?  all the other jquery events have a semicolon at the end.  As far as I know, the semicolon is optional... so given all the other events (at least in this area of the js) is using a semicolon, we should continue to do so.

Reply:
Not intentional will fix was part of moving things around or commenting. There are ~400 errors on that page in code analysis, so linting isn't doing me any good, it's a total mess, so new ones don't jump out. Outside of Discover, most pages have hundreds of linting errors and code style and general engineering flaws that I hope to address at some point, through a combination of slow refactors and in other areas total rewrites.

Reply:
Line 651 semicolon added back